src/scripts/game.js

```diff
@@ -207,6 +238,34 @@ class Game {
       this.beehiveSparkles.splice(this.beehiveSparkles.indexOf(object), 1);
     }
   }
+
+  updateHealth(points) {
+    this.health += points;
+    if (this.health <= 0) this.bee.capture();
+    if (this.health > 100) this.health = 100;
+    return this.health;
```

Comment:
Do you use the return value here?

Reply:
Good catch, at this point the return value is no longer used. Will clean up!